strings/Anagram.java

```diff
@@ -0,0 +1,34 @@
+import java.util.Scanner;
+public class Anagram {
+
+	public static void main(String[] args) {
+		Scanner scan=new Scanner(System.in);
+		String a=scan.nextLine();
+		String b=scan.nextLine();
+		 boolean ana=true;
+	      int[] u=new int[256];
+	      int[] v=new int[256];
+	      a=a.toLowerCase();
+	      b=b.toLowerCase();
+	
+	      for(char c : a.toCharArray())
+	      {
+	          int i=(int)c;
+	          u[i]++;
+	      }
+	       for(char c : b.toCharArray())
+	      {
+	          int i=(int)c;
+	          v[i]++;
+	      }
```

Comment:
What is going on here?

Reply:
* Here i have initialized 2 arrays u and v first for string a and second for b .
* Converted the strings to lowercase. 
* Then alternatively, while converting the char of the string to int(we'll get the ASCII value of that character) then 
  incrementing that particular index of the string by 1(for eg : if char is 'a' then we'll increment the 97th(ASCII value of 
  a) index of the array).
*  We'll follow these steps for both the strings (hence 2 for loops are used) . 
* At the end we'll check if both the arrays are equal then the strings are anagram of each other else not.